scifin/classifier/classifier.py

```diff
@@ -98,8 +102,8 @@ def dtw_distance(ts1, ts2, window=None, mode='abs', verbose=False):
     
     # Checks
     if not isinstance(ts1, ts.TimeSeries) and not isinstance(ts2, ts.TimeSeries):
-        raise AssertionError("Series have to be of type TimeSeries.")
-    if not isinstance(mode, str) and not mode in ('abs', 'square'):
+        raise TypeError("Series have to be of type TimeSeries.")
+    if (not isinstance(mode, str)) or (not mode in ('abs', 'square')):
```

Comment:
you can just use one conditional here, i.e. `if mode not in ('abs', 'square')`
PS: I would rather use `xx not in ..` than `not xx in ...`